### ISSUE TITLE
TypeError: n.relations.find(...) is undefined

### ISSUE THREAD

**Issue by txau** [huridocs/uwazidocs#4547]
Probably introduced in https://github.com/huridocs/uwazi/commit/84c33b7c35425f336997819a2b557aca730262d6 or https://github.com/huridocs/uwazi/commit/26815a59625f1b7c59b653027d48e29226eda008

/cc @konzz @grafitto @LaszloKecskes 

Certain combination of inheritance crashes the client on:

```
Uncaught (in promise) TypeError: n.relations.find(...) is undefined
    getSelectOptions formater.js:136
    getThesauriValues formater.js:356
    getThesauriValues formater.js:356
```

**Comment by txau** [comment]
related? https://github.com/huridocs/uwazi/issues/4548

**Comment by LaszloKecskes** [comment]
Based on an initial check and discussions with @konzz:

The two issues seem to be related. In both cases, what happens is:
- the failing entity A has relationship with another entity B in a **property**
- there is no connection between A and B in **connections** (i.e. no entry in the db for the two where the hub is the same). In Cejil, you can see this on B (specifically: pxi1pur6z1), which is successfully rendering, but indeed has no relationships to A.
- then, the lines introduced in https://github.com/huridocs/uwazi/commit/84c33b7c35425f336997819a2b557aca730262d6, where we try to use that information, fail.

In short, there seem to be missing connections from the database.

Question marks:
- we don't know why the connections are missing.
- I don't see a connection to template save. @txau, you've mentioned that this triggered when you were trying to remove a page from a template. Can you elaborate, so we can reproduce this angle?

Currently, I'm looking through the rendering pipeline to find a quick fix (by skipping missing information), but we should discuss a fix for the data as well.
